packages/frontend/src/helpers/customLogos.tsx

```diff
@@ -0,0 +1,15 @@
+import { createIcon } from '@chakra-ui/react'
+
+// have to create icon because images don't allow to modify svg content
+export const PipeIcon = createIcon({
```

Comment:
nit: would prefer this to be in its own `PipeIcon` tsx file in components folder instead of consolidating all logos into a single file. Reason is to keep with existing convention of 1 component = 1 folder.